src/main/java/org/junit/runners/ParentRunner.java

```diff
@@ -358,6 +358,7 @@ public Description getDescription() {
     public void run(final RunNotifier notifier) {
         EachTestNotifier testNotifier = new EachTestNotifier(notifier,
                 getDescription());
+        testNotifier.fireTestSuiteStarted();
```

Comment:
Well, since we can in theory report errors for the nested suites in `Parameterized` this is okay with me. I want to wait for the other maintainer to see if they have concerns.

I am curious to find the other maintainers plan on using `RunListener` for JUnit Lambda, and if so whether this change would work.